### ISSUE TITLE
not working in Grafana version 6.x 

### ISSUE THREAD

**Issue by ashoktmw** [mtanda/grafana-sumologic-datasource#32]
Error is not actually displayed in Front end.When i configure the datasource in Grafana and click "save and test", it should save the config and display the message Datasource successfully connected. (this is expected)
But, I'm not getting the datasource connection successful, rather i'm seeing it shows testing for long time.

**Comment by sanat1772** [comment]
Yes, this is coming for me too.

**Comment by sanat1772** [comment]
@mtanda Any clue?

**Comment by mtanda** [comment]
Sorry for late reply.
I use provisioning to setup datasource, I didn't notice the issue.

1.0.11 release may fix reported issue, please test.

**Comment by sanat1772** [comment]
Hey @mtanda Thanks for new version. But issue still persists, it is stuck on testing, and after following the same steps it is giving 403, but i am able to query sumologic via api call from server, but from grafana not happening.

**Comment by mtanda** [comment]
If possible, would you try to get error message from your browser's dev tools.
I can't replicate your issue in my env, so it is hard to investigate...

**Comment by sanat1772** [comment]
{
  "status" : 403,
  "id" : "<>",
  "code" : "forbidden",
  "message" : "This operation is not allowed for your account type."
}

@mtanda I am getting this error when i am testing.

**Comment by ashoktmw** [comment]
@mtanda , Thanks for working on this to get latest version ..Still I too getting same error.
Here are the steps I followed to install the plugin.

Step1: grafana-cli --pluginUrl https://github.com/mtanda/grafana-sumologic-datasource/releases/download/1.0.11/grafana-sumologic-datasource-1.0.11.zip plugins install mtanda-sumologic-datasource
Step2: Copy SRC folder in my plugin and restarted grafana .
![image](https://user-images.githubusercontent.com/58054382/79884322-e1d89280-83a9-11ea-9c6e-e27786d2ff01.png)
Any thing that we need to change in installation steps

**Comment by ashoktmw** [comment]
@mtanda , Its working after I realize the error in the backend..
When i use HTTP URL as "**https://api.us2.sumologic.com/api**" then its  connecting successfully. 
Once again! Thanks for the wonderful plugin

**Comment by mtanda** [comment]
@ashoktmw That's good.😄

**Comment by mtanda** [comment]
@sanat1772 According to doc, the account type is not allowed to use search job api.
https://help.sumologic.com/APIs/Troubleshooting-APIs/API-403-Error-This-operation-is-not-allowed-for-your-account-type
Is your account type is Enterprise?

**Comment by sanat1772** [comment]
@mtanda My account type is Sumo Logic Logs and Metrics Analytics Service - Professional Edition (Cloud Flex). Is it not enterprise?

**Comment by mtanda** [comment]
Sorry, I use Enterprise. I don't know Pro account can use search API.
Would you test to use API by curl.

https://help.sumologic.com/APIs/Search-Job-API/About-the-Search-Job-API

**Comment by sanat1772** [comment]
@mtanda Yes I am getting same error. I checked also, it is not available for professional account.

**Comment by michaelfward** [comment]
Hi,

I'm getting this error, but do have an enterprise account. 

I am seeing 202 when creating the job, and then a `DELETE` request without ever getting any data. 
<img width="696" alt="Screen Shot 2020-06-30 at 12 26 11 PM" src="https://user-images.githubusercontent.com/6685617/86168442-1022ce80-bacd-11ea-8f3f-342a62812962.png">

<img width="579" alt="Screen Shot 2020-06-30 at 12 28 19 PM" src="https://user-images.githubusercontent.com/6685617/86168559-3d6f7c80-bacd-11ea-9632-72cf7a13a5bf.png">

I'm using `grafana-server v7.0.5`

**Comment by ashoktmw** [comment]
@mtanda  - I see same issue again that in version 8, when I save and test the datasource connection it shows  testing for long time

**Comment by ashoktmw** [comment]
![image](https://user-images.githubusercontent.com/58054382/144476831-d9bd9683-272e-4a29-9115-a9febb2ce4ce.png)
